<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>461</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Размер мира</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbWorldSize">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="displayIntegerBase">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Визуализация</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbOutType">
       <item>
        <property name="text">
         <string>Консоль</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Черепаха</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Размер окна</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbWindowSize">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="displayIntegerBase">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Интервал между поколениями</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="dsbDelay"/>
     </item>
     <item>
      <widget class="QCheckBox" name="cbRandomInit">
       <property name="text">
        <string>Рандомная инициализация</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbCheckRepeat">
       <property name="text">
        <string>Проверять поколение на повтор</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbRun">
       <property name="text">
        <string>RUN</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>457</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
